src/components/popupWithConfirm.js

```diff
@@ -0,0 +1,29 @@
+import Popup from "./Popup.js";
+
```

Comment:
Needs correction: This file should be named PopupWithConfirm.js (starting with an upper case letter)